client/app/components/Blockquote/__tests__/Blockquote.spec.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+import { render, screen } from '@testing-library/react';
+
+import { Blockquote } from 'components/Blockquote';
+import React from 'react';
+
+const text = 'Quisque sit amet nulla sed erat tempus efficitur a id.';
+const author = 'Nullam ante';
+
+describe('Blockquote', () => {
+  describe('has no props', () => {
+    it('renders correctly', () => {
+      render(<Blockquote />);
+      expect(screen).not.toBeNull();
+    });
+  });
+
+  describe('has text prop', () => {
+    it('renders correctly with text prop', () => {
+      const { container, queryByText } = render(<Blockquote text={text} />);
+      const textSection = container.querySelector('.text');
+      expect(container).not.toBeNull();
```

Comment:
You can simplify this test by looking for the presence of the text. Instead of all these expects, you can have the following:


```
expect(queryByText(text)).toBeInTheDocument();
```